benchmark_results.md

```diff
@@ -0,0 +1,96 @@
+|   Java-Quarkus	|  simple ➙ default |  ✅ Mongo DB ➙ clean-before	|  1 	|  4 	|   2 min	|  17932 	|  17932/0 	|  26 ms 	|   149/sec	|
+|   Java-Quarkus	|  simple ➙ default |  ✅ Mongo DB ➙ clean-before	|  2 	|  4 	|   2 min	|  17750 	|  17750/0 	|  26 ms 	|   147/sec	|
+|   Java-Quarkus	|  simple ➙ default |  ✅ Mongo DB ➙ clean-before	|  5 	|  4 	|   2 min	|  15929 	|  15929/0 	|  29 ms 	|   132/sec	|
+|   Java-Quarkus	|  simple ➙ default |  ✅ Mongo DB ➙ clean-before	|  1 	|  8 	|   2 min	|  21394 	|  21394/0 	|  44 ms 	|   178/sec	|
+|   Java-Quarkus	|  simple ➙ default |  ✅ Mongo DB ➙ clean-before	|  2 	|  8 	|   2 min	|  21532 	|  21532/0 	|  44 ms 	|   179/sec	|
+|   Java-Quarkus	|  simple ➙ default |  ✅ Mongo DB ➙ clean-before	|  5 	|  8 	|   2 min	|  19577 	|  19577/0 	|  48 ms 	|   163/sec	|
+|   Java-Quarkus	|  simple ➙ default |  ✅ Mongo DB ➙ clean-before	|  1 	|  100 	|   2 min	|  25624 	|  25624/0 	|  469 ms 	|   212/sec	|
+|   Java-Quarkus	|  simple ➙ default |  ✅ Mongo DB ➙ clean-before	|  2 	|  100 	|   2 min	|  25525 	|  25525/0 	|  470 ms 	|   211/sec	|
+|   Java-Quarkus	|  simple ➙ default |  ✅ Mongo DB ➙ clean-before	|  5 	|  100 	|   2 min	|  23237 	|  23237/0 	|  519 ms 	|   191/sec	|
+|   Java-Quarkus	|  simple ➙ pool&workers=100 |  ✅ Mongo DB ➙ id-index	|  1 	|  1 	|   2 min	|  15214 	|  15214/0 	|  7 ms 	|   126/sec	|
+|   Java-Quarkus	|  simple ➙ pool&workers=100 |  ✅ Mongo DB ➙ id-index	|  2 	|  1 	|   2 min	|  10694 	|  10694/0 	|  10 ms 	|   89/sec	|
+|   Java-Quarkus	|  simple ➙ pool&workers=100 |  ✅ Mongo DB ➙ id-index	|  5 	|  1 	|   2 min	|  9223 	|  9223/0 	|  12 ms 	|   76/sec	|
+|   Java-Quarkus	|  simple ➙ pool&workers=100 |  ✅ Mongo DB ➙ id-index	|  1 	|  4 	|   2 min	|  41376 	|  41376/0 	|  11 ms 	|   344/sec	|
+|   Java-Quarkus	|  simple ➙ pool&workers=100 |  ✅ Mongo DB ➙ id-index	|  2 	|  4 	|   2 min	|  29132 	|  29132/0 	|  15 ms 	|   242/sec	|
+|   Java-Quarkus	|  simple ➙ pool&workers=100 |  ✅ Mongo DB ➙ id-index	|  5 	|  4 	|   2 min	|  20958 	|  20958/0 	|  22 ms 	|   174/sec	|
+|   Java-Quarkus	|  simple ➙ pool&workers=100 |  ✅ Mongo DB ➙ id-index	|  1 	|  8 	|   2 min	|  59936 	|  59936/0 	|  15 ms 	|   499/sec	|
+|   Java-Quarkus	|  simple ➙ pool&workers=100 |  ✅ Mongo DB ➙ id-index	|  2 	|  8 	|   2 min	|  46515 	|  46515/0 	|  20 ms 	|   387/sec	|
+|   Java-Quarkus	|  simple ➙ pool&workers=100 |  ✅ Mongo DB ➙ id-index	|  5 	|  8 	|   2 min	|  26142 	|  26142/0 	|  36 ms 	|   217/sec	|
+|   Java-Quarkus	|  simple ➙ pool&workers=100 |  ✅ Mongo DB ➙ id-index	|  1 	|  100 	|   2 min	|  117212 	|  117212/0 	|  102 ms 	|   971/sec	|
+|   Java-Quarkus	|  simple ➙ pool&workers=100 |  ✅ Mongo DB ➙ id-index	|  2 	|  100 	|   2 min	|  118294 	|  118294/0 	|  100 ms 	|   985/sec	|
+|   Java-Quarkus	|  simple ➙ pool&workers=100 |  ✅ Mongo DB ➙ id-index	|  5 	|  100 	|   2 min	|  47730 	|  47730/0 	|  250 ms 	|   397/sec	|
```

Comment:
Shouldn't we also add the csv file with the same results that can be downloaded to easily filter and extract data? I'm not sure we can do the same data manipulation from this Markdown table